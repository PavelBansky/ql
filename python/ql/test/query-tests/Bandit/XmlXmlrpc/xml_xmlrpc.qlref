Bandit/xml_xmlrpc.ql