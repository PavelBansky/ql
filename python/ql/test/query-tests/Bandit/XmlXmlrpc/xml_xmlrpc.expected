| Test.py:2:6:2:23 | ImportExpr | Using xmlrpclib to parse untrusted XML data is known to be vulnerable to XML attacks. |
